website/src/documentation/dsls/sql/extensions/windowing-and-triggering.md

```diff
@@ -18,7 +19,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Beam SQL: Windowing and triggering
+# Beam SQL extensions: Windowing and triggering
```

Comment:
similarly, not all pages in the Beam SQL extensions section have this prefix on the page title